src/FileFormats/CBF/write.jl

```diff
@@ -125,6 +130,56 @@ function _add_cones(
     return
 end
 
+function _add_cones(
+    data::_CBFDataStructure,
+    model::Model,
+    ::Type{F},
+    ::Type{S},
+) where {F<:MOI.VectorOfVariables,S}
+    for ci in MOI.get(model, MOI.ListOfConstraintIndices{F,S}())
+        f = MOI.get(model, MOI.ConstraintFunction(), ci)
+        is_variable_cone = true
+        for (i, xi) in enumerate(f.variables)
+            if xi in data.variables_with_domain
+                is_variable_cone = false
+                break
+            elseif xi.value != f.variables[1].value + i - 1
+                is_variable_cone = false
+                break
+            end
+            push!(data.variables_with_domain, xi)
+        end
+        str = _cone_string(data, S)
+        if !is_variable_cone
+            _add_function(data, f, S)
+            set = MOI.get(model, MOI.ConstraintSet(), ci)
+            push!(data.cones, (str, MOI.dimension(set)))
+        else
+            push!(data.variable_cones, (f.variables, str))
+        end
+    end
+    return
+end
+
+function _add_cones(
+    data::_CBFDataStructure,
+    model::Model,
+    ::Type{F},
+    ::Type{S},
+) where {
+    F<:MOI.VectorOfVariables,
+    S<:Union{MOI.ExponentialCone,MOI.DualExponentialCone},
+}
+    # The Exponential cone in MOI and CBF are reversed. Instead of dealing with
```

Comment:
I'd prefer if we say it's not supported for now. Is it the same permutation as https://github.com/jump-dev/ECOS.jl/blob/master/src/MOI_wrapper/permuted_exponential_cone.jl

Reply:
It should be the same as the Mosek exponential cone I guess. If you want it to be supported, you can add the set and bridge in this module. Then we will be able to simplify MosekTools by using this set and adding the bridge in the `ListOfNonstandardBridges`

Reply:
It's not quite the same permutation as ECOS. ECOS is `1, 3, 2`, where this is `3, 2, 1`.

Reply:
Yes, that's what I remembered, Mosek and ECOS have a different one ^^